setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup
+
+import elife_api
+
+setup(
+    name='elife_api',
+    version=elife_api.__version__,
+    description='Utility for accessing eLife API-Raml JSON schemas in Python.',
+    packages=['elife_api'],
+    license='MIT',
+    url='https://github.com/elifesciences/api-raml',
+    maintainer='eLife Sciences Publications Ltd.',
+    maintainer_email='py@elifesciences.org',
```

Comment:
Does that exist? Could be `tech-team@elifesciences.org`?

Reply:
I got it from https://github.com/elifesciences/elife-tools/blob/05def8780aaf7ffb247c409ad77d62d186f1abdd/setup.py#L20 will change to tech-team@elifesciences.org

Reply:
After testing `py@elifesciences.org` I got a delivery failure notification, so I think its safe to assume its no longer working.

It has now been changed:
https://github.com/elifesciences/api-raml/blob/df57ae9e9db08516ac73ec44d1d9509f15f4b820/setup.py#L13